app/templates/main/request-received.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+{% block beforeContent %}
+{{ super() }}
+{% endblock %}
```

Comment:
Can we still drop this or did it break something?

Reply:
Yes  it breaking up things, we need this.

Reply:
Are you just removing `{{ super() }}` or the entire block?